spectator-reg-tdigest/src/main/java/com/netflix/spectator/tdigest/StreamTDigestReader.java

```diff
@@ -27,17 +29,19 @@
  */
 public class StreamTDigestReader implements TDigestReader {
 
+  private final Json json;
   private final DataInputStream in;
 
   private final byte[] buf = new byte[TDigestWriter.BUFFER_SIZE];
 
   /** Create a new instance. */
-  public StreamTDigestReader(InputStream in) {
-    this.in = new DataInputStream(in);
+  public StreamTDigestReader(Registry registry, InputStream in) {
```

Comment:
This class is public, the `Json` object is package private. I didn't want to expose that class outside the package.